snp_automate_2022/README.md

```diff
@@ -0,0 +1,75 @@
+# SNP Implementation at Automate 2022
+
+This demo uses a Motoman HC10 mounted on a table with an Intel RealSense camera to reconstruct the surface of an arbitrary part and generate motion plans for polishing the part in a raster pattern
+
+## Build Setup
+
+1. Follow the [build setup instructions](../README.md) for the main repository
+
+1. Clone the application-specific ROS2 dependencies into the same workspace
+    ```
+    cd <snpd_workspace>
+    vcs import src < snp_automate_2022/dependencies.repos
+    ```
+
+## Build
+
+```
+colcon build --cmake-args -DTESSERACT_BUILD_FCL=OFF
+```
+
+## ROS1 Hardware Interface Software Installation
+
+Install the requisite ROS1 driver software to run the application on hardware
+
+> Note: this step is not required to run the application in simulation only
+
+1. Build the ros1_bridge
+    - Create a new workspace, clone this branch of the ros1_bridge repo
+      ```bash
+      git clone -b action_bridge https://github.com/ipa-hsd/ros1_bridge.git
+      ```
+    - Source the both ROS distros
+      ```bash
+      source /opt/ros/foxy/setup.bash
+      source /opt/ros/noetic/setup.bash
+      ```
+    - Build the bridge
+      ```bash
+      colcon build --symlink-install --packages-select ros1_bridge --cmake-force-configure
+      ```
+
+1. Build the ROS1 workspace
+    - Create a new workspace, clone this commit of the motoman repo
+      ```bash
+      git clone -b 63c94ec https://github.com/ros-industrial/motoman.git
+      ```
+    - Source the ROS1 installation
+      ```bash
+      source /opt/ros/noetic/setup.bash
+      ```
+    - Build the repo
+      ```bash
+      catkin build
+      ```
+
+## Running the system
+### Simulation
+```bash
+ros2 launch snp_automate_2022 start.launch.xml
+```
+
+### On Hardware
+1. Start the ROS1 launch file
+    ```bash
+    roslaunch motoman_hc10_support robot_interface_streaming_hc10.launch robot_ip:=192.168.1.31 controller:=yrc1000
+    ```
+1. Run the bridge
+    ```bash
+    source install/setup.bash
+    ros2 run ros1_bridge dynamic_bridge --bridge-all-1to2-topics
+    ```
+1. Start the ROS2 launch file
+    ```bash
+    ros2 launch snp_automate_2022 start.launch.xml sim_robot:=false sim_vision:=false
+    ```
```

Comment:
I think you should specify that running the hardware is in three separate terminals and also specify which workspace to navigate to and sourcing for each. It might be clear to us, but I think with the potential range of target users that would be helpful